### PR TITLE
Chainweb stream client (broken branch)

### PR DESCRIPTION
**Pull request by Takadenoshi**
WIP chainweb-stream-client branch

Still need to extend the README a bit, but the approach and code should be mostly ready for review

Backend: https://github.com/kadena-io/chainweb-sse/

Will open a PR on the account tracker to show demo usage

### CODE REVIEW

**Comment by Takadenoshi** [comment]
Changes:
- chainweb-stream-client example script - src/example/index.js
- added data event for both unconfirmed and confirmed transactions
- added cross chain ID and Account to ChainwebAccountData interface (new in chainweb-data)
- changed types to make interface naming consistent; merged .meta and base transaction types
- Added eventsource polyfill for unsupported environments (outdated bro…
- fix debug event, payload was lost along the way
- fix/pin @types/eventsource to 1.1.10 - 1.1.11 breaks for TS < 5.0

File: packages/libs/chainweb-stream-client/src/tests/index.test.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+// import { x } from '..';
```
Comment (EnoF) [inline packages/libs/chainweb-stream-client/src/tests/index.test.ts:1]: Did you intend to add this empty test?

**Review by EnoF — APPROVED** [review]

**Review by ash-vd — APPROVED** [review]

File: packages/libs/chainweb-stream-client/src/tests/index.test.ts (hunk 1 above)
Comment (Takadenoshi) [reply packages/libs/chainweb-stream-client/src/tests/index.test.ts]: Nope! Fixed, thanks

**Comment by Takadenoshi** [comment]
Issues with @react/type incorrect resolution persisted, seems to be rush cache related somehow but still haven't pinned the issue.

Opened a new PR for a fresh from-master branch here:

 https://github.com/kadena-community/kadena.js/pull/249

**Pull request closed without merging**